### PR TITLE
Prototype apache docker image built. Configuration changes will be re…

### PR DESCRIPTION
**Pull request by SHulce**
…quired later.

Basic configuration ready to go.

### CODE REVIEW

File: docker-cadg-apache/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,18 @@
+### Create the AOI Apache web frontend image
+
+# Get base apache image from Docker Hub
+FROM httpd:2.4
+
+# Install VIM editor for configuration changes inside the machine 
+# This is not the recommended way to do it, its best to configure the file first.
+# However, it may be necessary to modify depending on the server.
+RUN apt-get update
+RUN apt-get install vim -y
```
Comment (kchoadley) [inline docker-cadg-apache/Dockerfile]: and nano, please!

File: docker-cadg-apache/README.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# CADG Apache Docker Image
+
+This utilizes version 2.4 of the Apache httpd server. The configuration file is extremely basic at present and will need to be built. As such, this has been tagged as prototype for the time being.
+
+To build locally
+
+docker build -t image-name:tag .
```
Comment (kchoadley) [inline docker-cadg-apache/README.md]: Wrap in triple backticks.
Change to "docker build -t cadg-apache:tag ."

File: docker-cadg-apache/README.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+Note: Replace image-name:tag with your desired nomenclature. Also note that you can pull the current prototype image from the docker hub with:
+
+docker pull os-wasabi/cadg-apache:prototype
```
Comment (kchoadley) [inline docker-cadg-apache/README.md]: Wrap in triple backticks.

File: docker-cadg-apache/README.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+
+
+Run Directions:
+To run this container you will need to provide it port information as well as a name. As below:
+
+docker run -d --name container-name -p 8080:80 cadg-apache:prototype
```
Comment (kchoadley) [inline docker-cadg-apache/README.md]: Wrap in triple backticks.

File: docker-cadg-apache/Dockerfile (hunk 1 above)
Comment (SHulce) [reply docker-cadg-apache/Dockerfile]: No nano for you! 


...okay i added it

File: docker-cadg-apache/README.md (hunk 2 above)
Comment (SHulce) [reply docker-cadg-apache/README.md]: :+1:

File: docker-cadg-apache/README.md (hunk 3 above)
Comment (SHulce) [reply docker-cadg-apache/README.md]: :+1:

File: docker-cadg-apache/README.md (hunk 4 above)
Comment (SHulce) [reply docker-cadg-apache/README.md]: :+1:

**Comment by SHulce** [comment]
PR Review Modifications push re: issue-115 on Taiga.

**Pull request merged**